source/includes/steps-starting-vsce-paste-string.yaml

```diff
@@ -24,21 +24,61 @@ content: |
     :manual:`Connection String URI Format
     </reference/connection-string>`.
 ---
-source:
-  file: steps-source-command-palette.yaml
-  ref: vsce-command-palette
-ref: vsce-connect-command-palette
----
-title: "From the Command Palette, select 
-      :guilabel:`MongoDB: Connect with Connection String...`."
+title: "In |vscode-short|, open the Command Palette from the |vsce| 
+       *Overview* page, from the :guilabel:`MongoDB` view in the 
+       :guilabel:`Activity Bar`, or directly to paste the connection 
+       string."
 level: 4
-ref: vsce-connect-connect
-content: |
+ref: vsce-open-overview-page
+content: | 
 
-  .. include:: /includes/admonitions/tip-mongodb-command-palette.rst
+  .. tabs:: 
 
----
+     .. tab:: Open from Overview Page 
+        :tabid: mdboverview
+
+        a. Perform the action described in the following table for your 
+           operating system and method:
+
+           .. include:: /includes/list-tables/command-palette-options.rst
+        
+        #. From the Command Palette, select :guilabel:`MongoDB: Open 
+           Overview Page`.
+
+           .. include:: /includes/admonitions/tip-mongodb-command-palette.rst
+
+        #. In the *Overview* page, click :guilabel:`Connect` under the 
+           :guilabel:`Connect with Connection String`.
 
+     .. tab:: Open from MongoDB View
+        :tabid: mdbview 
+
+        a. Expand the :guilabel:`Connections` pane in the left 
+           navigation if it is collapsed.
+           
+        #. Click the :guilabel:`More Actions` menu (:guilabel:`...`), 
```

Comment:
I think rather than providing two options here, it might be better to simplify and only mention the second :guilabel:`Add MongoDB Connection with Connection String...` option. I think this option is the easier of the two, and it might help to provide a bit more direction rather than have the user make a choice.